Assets/_BForBoss/_Core/Scripts/Managers/WeaponRaceCourseWorldManager.cs

```diff
@@ -1,82 +0,0 @@
-using System;
-using Perigon.Analytics;
-using Perigon.Leaderboard;
-using Perigon.Utility;
-using Sirenix.OdinInspector;
-using UnityEngine;
-using UnityEngine.Rendering;
-
-namespace BForBoss
-{
-    public class WeaponRaceCourseWorldManager: BaseWorldManager
```

Comment:
This being used in `WeaponRaceCourse.unity`, `MS2_RaceCourse.unity`, `MatTestScene.unity`

Reply:
All the race course senes are removed from the scene selection.